mipengine/node_tasks_DTOs.py

```diff
@@ -1,87 +1,87 @@
-from dataclasses import dataclass
+import enum
 from typing import Any
 from typing import Dict
 from typing import List
 from typing import Union
+from warnings import warn
 
-from dataclasses_json import dataclass_json
+from pydantic import (
+    BaseModel,
+    validator,
+)
 
 
-@dataclass_json
-@dataclass
-class ColumnInfo:
-    name: str
-    data_type: str
+# ~~~~~~~~~~~~~~~~~~~~ Enums ~~~~~~~~~~~~~~~~~~~~ #
+
+
+class DBDataType(enum.Enum):
```

Comment:
As discussed in the call, this could be completely removed in favor of DType and the unification of the datatypes that @jassak started.

So this could be completely removed and use DType instead. I don't know how this could work since these Enums are a bit "special".

This [method](https://github.com/madgik/MIP-Engine/blob/717fcad868d8ff625e8e5ce8e30446ef63705b8f/mipengine/node/monetdb_interface/common_actions.py#L250) could also be completely removed in favor of DType.

This [method](https://github.com/madgik/MIP-Engine/blob/717fcad868d8ff625e8e5ce8e30446ef63705b8f/mipengine/node/monetdb_interface/common_actions.py#L267) should be also completely removed but I am not sure this is possible, since MonetDB returns a column with schema "varchar(50)" as "varchar" so there is a problem with the conversion there.

Leaving it with you and Jason to solve.